cmd/cluster/context.go

```diff
@@ -896,3 +924,18 @@ func (data *contextData) printClusterHeader(w io.Writer) {
 	fmt.Fprintln(w, clusterHeader)
 	fmt.Fprintln(w, strings.Repeat("=", len(clusterHeader)))
 }
+
+func printMigrationStatus(data *contextData, w io.Writer) {
+	var name string = "Migration Status"
+	fmt.Fprintln(w, "\n"+delimiter+name)
+
+	if data.SdnToOvnMigration == nil {
+		fmt.Fprintln(w, "No active migrations")
```

Comment:
Similar to above, `No active migration`, targeting this code to _just_ the SDN to OVN migration